### ISSUE TITLE
ps to field scale problem

### ISSUE THREAD

**Issue by joellebegin** [joellebegin/Power-Spectra#17]
New pspec to field works very nicely.

When I give the generated field to my power spectrum code, it returns the shape and peak location _perfectly_, but the power scale is wildly off (like, many many orders of magnitude). 

Have no idea where the issue is. Old pspec to field code did this as well, I just didn't care much because it was qualitative but now we actually need this to work properly.

**Comment by acliu** [comment]
Volume factors, maybe?

**Comment by joellebegin** [comment]
Yeahh that's what I've been thinking. I figure I  have to multiply by the volume somewhere in there (since in computing the power spectrum you divide by the volume) so I'm just multiplying by volume wherever it seems reasonable to see if changes things. 

No luck yet.

**Comment by acliu** [comment]
I call that "debugging by Monte Carlo". Very dangerous. Not recommended.

If P(k) ~ < |T-tilde (k)|^2 > / V, then the standard deviation is sqrt(V P(k)).

**Comment by joellebegin** [comment]
Yeah, that's I what I have! still doesn't give the right scale though

**Comment by acliu** [comment]
Fascinating.

**Comment by joellebegin** [comment]
Yeah :( I'm sure it's just some units not being converted properly or something; will keep looking!
